src/Classes/TradeQuery.lua

```diff
@@ -495,30 +555,47 @@ function TradeQueryClass:PriceItemRowDisplay(str_cnt, slotTbl, top_pane_alignmen
 				end,
 				{
 					callbackQueryId = function(queryId)
-						controls["uri"..context.str_cnt]:SetText("https://www.pathofexile.com/trade/search/".. self.pbRealm .. self.pbLeague:gsub(" ", "+") .."/".. queryId)
+						local url = self.tradeQueryRequests:buildUrl("https://www.pathofexile.com/trade/search", self.pbRealm, self.pbLeague, queryId)
+						controls["uri"..context.str_cnt]:SetText(url, true)
 					end
 				}
 			)
 		end)
 	end)
-	controls["bestButton"..str_cnt].enabled = function() return main.POESESSID ~= "" end
+	-- controls["bestButton"..str_cnt].enabled = function() return main.POESESSID ~= "" end
```

Comment:
Is this meant to be commented out / deleted?

Reply:
Yes, leftover from testing.